lib/filesystem.nix

```diff
@@ -34,19 +46,19 @@ in
   */
   pathType =
     builtins.readFileType or
-    # Nix <2.14 compatibility shim
-    (path:
-      if ! pathExists path
-      # Fail irrecoverably to mimic the historic behavior of this function and
-      # the new builtins.readFileType
-      then abort "lib.filesystem.pathType: Path ${toString path} does not exist."
-      # The filesystem root is the only path where `dirOf / == /` and
-      # `baseNameOf /` is not valid. We can detect this and directly return
-      # "directory", since we know the filesystem root can't be anything else.
-      else if dirOf path == path
-      then "directory"
-      else (readDir (dirOf path)).${baseNameOf path}
-    );
+      # Nix <2.14 compatibility shim
```

Comment:
Don't run code formatters unless it's really necessary (which I don't think is the case here).